pluginkit/evaluation_orchestrator.go

```diff
@@ -30,20 +30,18 @@ type EvaluationOrchestrator struct {
 
 type DataLoader func(*config.Config) (interface{}, error)
 
-func NewEvaluationOrchestrator(pluginName string, loader DataLoader, requiredVars []string) *EvaluationOrchestrator {
```

Comment:
Is this removal better? I get the one-off `Add*` functions. Does allow for more `Add*` functions to be added later and you don't have to go edit all usages of the constructor. Thinking out loud.

Reply:
Yeah, I am starting to hate all of our constructors.

Reply:
I removed the NewChange constructor as well